packages/terra-form-select/tests/wdio/select-spec.js

```diff
@@ -344,7 +358,7 @@ Terra.describeViewports('Select', ['tiny'], () => {
       });
 
       it('dropdown should be focused', async () => {
-        (await browser.hasFocus('#terra-select-menu')).should.be.true;
+        (await browser.hasFocus('[class*=menu]')).should.be.true;
```

Comment:
Wildcard selectors can be flaky if anything else on the page has a similar class name. I think this is probably safe in this isolated test, if we wanted to be safe we could potentially introduce a data attribute similar to the frame. (Which I just noticed says combobox, but I think they all do)

https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/src/single/Frame.jsx#L489